plugins/kluster-capacity.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kluster-capacity
+spec:
+  version: v0.1.1
+  homepage: https://github.com/k-cloud-labs/kluster-capacity
+  shortDescription: Cluster capacity analysis tool.
```

Comment:
What do you think of "Scheduler simulation for capacity analysis"?

We already have plugins like "view-capacity", so this can stand out better in the list?

Reply:
good, i have updated it, thx